Gems/Multiplayer/Code/Source/Components/MultiplayerComponentRegistry.cpp

```diff
@@ -70,8 +75,31 @@ namespace Multiplayer
         return nullComponentData;
     }
 
+    AZ::HashValue64 MultiplayerComponentRegistry::GetMultiplayerComponentVersionHolisticHash() const
+    {
+        return m_componentVersionHash;
+    }
+
+    bool MultiplayerComponentRegistry::FindComponentVersionHashByName(const AZ::Name& multiplayerComponentName, AZ::HashValue64& hash) const
+    {
+        const auto it = m_componentVersionHashes.find(multiplayerComponentName);
+        if (it != m_componentVersionHashes.end())
+        {
+            hash = it->second;
+            return true;
+        }
+
+        return false;
+    }
+
+    const AZStd::unordered_map<AZ::Name, AZ::HashValue64>& MultiplayerComponentRegistry::GetMultiplayerComponentVersionHashes() const
+    {
+        return m_componentVersionHashes;
+    }
+
     void MultiplayerComponentRegistry::Reset()
```

Comment:
Should this clear `m_componentVersionHash `as well?

Reply:
thank you!